docs/customization.rst

```diff
@@ -56,6 +56,26 @@ To set a new theme, do
 
    $ $XONSH_COLOR_STYLE='<theme name>'
 
+Registering custom styles
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you aren't happy with the styles provided by us (and ``pygments``), you can create and register custom styles.
+
+To do so, add something similar to your ``.xonshrc``:
+
+.. code-block:: python
+
+   from xonsh.pyghooks import register_custom_style
```

Comment:
The registration function should probably be moved to `xonsh.tools`

Reply:
That was my original intention, but it would introduce circular dependency between `xonsh.pyghooks` and `xonsh.tools`. Can you suggest a way how can I avoid that?

Reply:
You can import pyghooks (if pygments is available) inside of the function itself.  I think that there are other functions in tools which do the same thing.